autoload/importjs.vim

```diff
@@ -251,17 +258,18 @@ function! importjs#Init()
     let s:job_executable='importjs'
   endif
 
-  " neovim
   if exists("*jobstart")
+    " neovim
     let s:job = jobstart([s:job_executable, 'start', '--parent-pid', getpid()], s:callbacks)
-  endif
-
-  " vim
-  if exists("*job_start")
+  elseif exists("*job_start")
```

Comment:
This is unrelated, but I decided to join these two blocks as only one should be used.